bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/altair/AltairParams.java

```diff
@@ -44,14 +44,7 @@ public String displayValue() {
         }
 
         public String sequenceValue() {
-            return _displayValue;
-        }
-
-        /**
-         * Return a Wavelength by index *
-         */
-        static public Wavelength getWavelengthByIndex(int index) {
-            return SpTypeUtil.valueOf(Wavelength.class, index, DEFAULT);
+            return name();
```

Comment:
I really don't like this convention because it means that we can no longer rename-refactor the identifiers without changing the meaning of the program. It is in effect a very subtle kind of reflection. So I would prefer a distinct string field in this case and others like it.

I realize that there's lots of this kind of thing in OCS, but we should fix it when we encounter it.

Does this make sense?